### ISSUE TITLE
Tests failures: test_luacheck, test_website_data, test_image_css

### ISSUE THREAD

**Issue by Ygrex** [luakit/luakit#601]
building from the latest 01eccb0aa77335fb4d15af53f6cdc71192e222fa with:
```
$ make DEVELOPMENT_PATHS=1 LUA_PKG_NAME=lua5.1 clean run-tests
```
fails three tests:
1.
```
FAIL style/test_luacheck / test_luacheck
  tests/style/test_luacheck.lua:103: Luacheck messages:
    lib/select_wm.lua:70: (22:25):           shadowing upvalue argument 'left' on line 65
    lib/select_wm.lua:71: (22:26):           shadowing upvalue argument 'right' on line 65
```
2.
```
FAIL async/test_clib_luakit / test_website_data
  Timed out while waiting
    interval was 1000msec
    
  Traceback:
  (1) tests/async/test_clib_luakit.lua:104 in function [anonymous]
```
3.
```
FAIL async/test_image_css / test_image_css
  tests/async/test_image_css.lua:26: bad argument #2 to 'format' (string expected, got table)
  Traceback:
  (1) tests/async/test_image_css.lua:26 in function view_wait_for_status
  (2) tests/async/test_image_css.lua:44 in function [anonymous]
```

**Comment by aidanholm** [comment]
I've fixed 1) but can't reproduce 2) or 3)

**Comment by aidanholm** [comment]
2) is probably just random slowness; 3) I'm not sure about; are you using luajit or regular lua?

**Comment by Ygrex** [comment]
I am on regular lua5.1

now building with f7a3a23865bd2adcf8f40de1065ac31040c21c4d gives me the only failure:
```
FAIL style/test_luacheck / test_luacheck
  tests/style/test_luacheck.lua:103: Luacheck messages:
    lib/session.lua:198: (15:15):           shadowing upvalue argument 'w' on line 184
```

original ones are all gone

**Comment by Ygrex** [comment]
thank you, fixed in f8b566c8ee533a23f206cd57357cdee20fda3642

**Comment by aidanholm** [comment]
Ok, great!
